pkg/minikube/machine/client_linux.go

```diff
@@ -41,11 +44,23 @@ func getKVMDriver(rawDriver []byte) (drivers.Driver, error) {
 	return driver, nil
 }
 
+func getLocalDriver(rawDriver []byte) (drivers.Driver, error) {
+	var driver drivers.Driver
+	driver = &local.Driver{}
+	if err := json.Unmarshal(rawDriver, &driver); err != nil {
+		return nil, errors.Wrap(err, "Error unmarshalling local driver")
+	}
+	return driver, nil
+}
+
 // StartDriver starts the desired machine driver if necessary.
 func registerDriver(driverName string) {
 	switch driverName {
 	case "virtualbox":
 		plugin.RegisterDriver(virtualbox.NewDriver("", ""))
+	case "local":
```

Comment:
We shouldn't support the RPC-separate binary client for the local driver.  The LocalClient allows the libmachine API to access the driver directly, and this should be the only way that the local client is used.  So no need to register the driver here.